test/scaffold/start.unit.js

```diff
@@ -163,4 +163,104 @@ describe('#start', function() {
       });
     });
   });
+  describe('#spawnChildProcess', function() {
+
+    it('should build the appropriate arguments to spawn a child process', function() {
+      var child = {
+        unref: function() {}
+      };
+      var _process = {
+        exit: function() {},
+        env: {
+          __bitcore_node: false
+        },
+        argv: [
+          'node',
+          'bitcore-node'
+        ],
+        cwd: 'cwd',
+        pid: 999,
+        execPath: '/tmp'
+      };
+      var fd = {};
+      var spawn = sinon.stub().returns(child);
+      var openSync = sinon.stub().returns(fd);
+      var spawnChildProcess = proxyquire('../../lib/scaffold/start', {
+        fs: {
+          openSync: openSync
+        },
+        child_process: {
+          spawn: spawn
+        }
+      }).spawnChildProcess;
+
+      spawnChildProcess('/tmp', _process);
+
+      spawn.callCount.should.equal(1);
+      spawn.args[0][0].should.equal(_process.execPath);
+      var expected = [].concat(_process.argv);
+      expected.shift();
+      spawn.args[0][1].should.deep.equal(expected);
+      var cp_opt = {
+        stdio: ['ignore', fd, fd],
+        env: _process.env,
+        cwd: _process.cwd,
```

Comment:
The docs for spawn mention string: https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options